server/routes/users/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+
+const User = require('../../../database/models/User');
+
+router.get('/profile', (req, res) => {
+  let id = req.query.user;
+
+  User.where({ 'id' : id }).fetch()
+    .then(fetched => {
+
+      fetched = fetched.attributes;
+      res.json(fetched);
+    })
+    .catch(err => {
+      res.json({
+        code : '400: Bad Request',
+        error : 'Invalid User ID'
+      })
+    })
+});
+
+router.put('/users', (req, res) => {
+  const id = req.query.user;
+  const userData = req.body;
```

Comment:
This route is incomplete and the way that it is, it'll hang the server. At least return a `res.json` that says that the route isn't finished yet.